app/src/main/java/cm/aptoide/pt/abtesting/AbTestCacheValidator.java

```diff
@@ -0,0 +1,35 @@
+package cm.aptoide.pt.abtesting;
+
+import java.util.HashMap;
+
+public class AbTestCacheValidator {
+
+  private HashMap<String, ExperimentModel> localCache;
+
+  public AbTestCacheValidator(HashMap<String, ExperimentModel> localCache) {
+    this.localCache = localCache;
+  }
+
+  public boolean validateCache(String id) {
```

Comment:
I’d call is cacheValidated, how it’s called right now looks like a void method

Reply:
fixed